custom/_legacy/pact/templates/pact/patient/pactpatient_providers.html

```diff
@@ -240,7 +240,7 @@
                 self.is_loading(true);
                 self.showSave(false);
                 var arr = self.selected_providers();
-                var csrftoken = $.cookie('csrftoken');
+                var csrftoken = $("#csrfTokenContainer").val();
```

Comment:
Agreed that that is what it looks like from code. To confirm, I checked for the view that renders it [here](https://www.commcarehq.org/a/pact/reports/custom/patient/?patient_id=0bd56681dce2b7a5140f75266d20880e&view=providers) and it had the container.